.azure-pipelines.yml

```diff
@@ -13,6 +13,7 @@ schedules:
 
 variables:
   GRADLE_USER_HOME: $(Pipeline.Workspace)/.gradle
+  ORT_DATA_DIR: $(Pipeline.Workspace)/.ort
```

Comment:
Oh, not strictly related to this change, but it just occurs to me that you set `.ort` and `.gradle` relative to the workspace root, i.e. the location where ORT is checked out, correct? Because also when building locally, Gradle creates a `.gradle` directory in the ORT project root. Doesn't it potentially cause problems to also move the contens that usually go to `$USER_HOME/.gradle` to that directory?

Reply:
BTW, on the general topic of Gradle caching, are you aware of the [shareable read-only dependency cache](https://docs.gradle.org/6.2/release-notes.html#shared-dependency-cache) introduced in Gradle 6.2?

Reply:
You're right, I just followed the Azure examples there. Luckily there is no overlap inside the directory and it causes no issue. But I think I'll make a commit latter to rename this directory to `.gradle-user-home` or something similar, or check if the default location works, despite all cache examples moving the cached directories into the working dir, because I don't see why it shouldn't.

Reply:
> check if the default location works, despite all cache examples

👍